src/util/FactorUtil.js

```diff
@@ -194,8 +194,7 @@ function (Okta, TimeUtil) {
   };
 
   fn.getFactorDescription = function (provider, factorType) {
-    var key = factorData[fn.getFactorName.apply(this, [provider, factorType])].description;
```

Comment:
curious why made this refactor?

Reply:
There was a request to not have a description for password, but we could not just pass '' or else we would see an i18nERROR[] (or something similar to that), since we are passing a non existent message key to Okta.loc.

Thus we need to move the definition up a stack so that we either pass a valid Okta.loc message, or empty string there.

Reply:
- I didn't see `password` has added into this factor list, did I miss anything?
- you could also do
```
var key = ....description`
return _.isEmpty(key) ? '' : okta.loc(key, 'login');
```
- the main question is `okta.loc` has to be invoke after language bundle loaded. I'm not user your refactor would break the sequencing or not.